README.md

```diff
@@ -7,6 +7,11 @@ If you use *Metabolic Atlas* in your scientific work, please cite:
 >
 > Robinson, J., et al, 2020. _An atlas of human metabolism_. Science Signaling 13 [doi:10.1126/scisignal.aaz1482 ](https://doi.org/10.1126/scisignal.aaz1482)
 
+## About Metabolic Atlas
+Metabolic Atlas is a web platform integrating open-source genome scale metabolic models (GEMs) for easy browsing and analysis. The goal is to collect many curated GEMs, and to bring these models closer to [FAIR principles](https://en.wikipedia.org/wiki/FAIR_data). The webiste provides visualisations and comparisons of the GEMs, and links to resourcess, algorithms, other databases, and more general software applications. In short, the vision is to create a one-stop-shop for everything metabolism related. 
+
+## Contributing
+If you would like to contribute to the project, for example if you have ideas for new features, discovered a bug or if you would like to improve the code base, please have a look at [CONTRIBUTING.md](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/.github/CONTRIBUTING.md).
```

Comment:
I would like this section to be even more encouraging. I've added a suggestion at the end. If you have any phrases/ideas to encourage contributions, maybe from other projects, please adopt them.
```suggestion
If you would like to contribute to the project, for example if you have ideas for new features, discovered a bug or if you would like to improve the code base, please have a look at [CONTRIBUTING.md](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/.github/CONTRIBUTING.md). We are looking forward to your ideas and contributions.
```

Reply:
I'm not really sure about this, "we" is not something used elsewhere in the text (or repo in general?). Maybe something like "All ideas and contributions are highly appreciated" or similar.

Reply:
> I'm not really sure about this, "we" is not something used elsewhere in the text (or repo in general?). Maybe something like "All ideas and contributions are highly appreciated" or similar.

I prefer more neutral tones so I think it's fine without any addition, but this is a very personal preference. If we are to add something I like your suggestion of the third-person narrative.

Reply:
Added a sentence  in ea44798bb04f5a20bde19f324630bde1f862693a

Reply:
Looks great!